lib/ace/ext-beautify.js

```diff
@@ -0,0 +1,334 @@
+ace.define("ace/ext/beautify/php_rules",["require","exports","module","ace/token_iterator"], function(require, exports, module) {
```

Comment:
Removed all but one of the extensions, replaced remaining with minified versions.